### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#304] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152321857-6e3ddfcc-0f94-4461-b37c-3588fb70f819.png)

**Comment by AxnNxs** [comment]
Caso Base (per n = 1):
2*1 - 1 = 1^2     =>    1 = 1

Passo induttivo (per n >= 1):
Ipotesi induttiva:     ∑ (2n - 1) = n^2
                              i = 1 a n
                              
P(n+1):                    ∑ (2n - 1) = (n + 1)^2
                              i = 1 a n+1
                              
Sostituendo P(n) in P(n +1):
      ∑ (2n - 1) + (2(n+1) - 1)= (n + 1)^2
   i = 1 a n
   
Da cui:
n^2 + 2n +1 = (n + 1)^2
(n+1)*(n+1) = (n + 1) ^ 2

Q.E.D.

**Comment by LuginiAndrea** [comment]
Dimostrazione col principio del minimo numero
C.B: $$n = 1, \sum_{i = 1}^{n} = n^2 \implies 2 - 1 = 1$$
Per assurdo, ipotizziamo che $m > 1$ sia il minimo numero per il quale la nostra ipotesi non vale, quindi
$$\sum_{i = 1}^m 2i - 1 \neq m^2 \implies \sum_{i = 1}^{m-1} 2i - 1 + 2m - 1\neq m^2 \iff \left(m-1\right)^2 +2m - 1 \neq m^2$$
$$\iff m^2 - 2m + 1 + 2m - 1 \neq m^2 \iff m^2 \neq m^2 \textrm{ Impossibile }$$
Il che implica che $m$ non è il minimo numero per il quale ciò vale: ne segue che $\nexists m$
